src/routes/write-exercise/WriteExerciseScreen.tsx

```diff
@@ -44,23 +45,31 @@ const WriteExerciseScreen = ({ navigation, route }: WriteExerciseScreenPropsType
   const { trainingSet, trainingSetId, disciplineTitle } = extraParams
   const [isLoading, setIsLoading] = useState(true)
   const [currentDocumentNumber, setCurrentDocumentNumber] = useState(0)
-
-  let documents = useLoadDocuments(trainingSetId).data
+  const [docs, setDocs] = useState<DocumentType[]>([])
+  const documents = useLoadDocuments(trainingSetId).data
```

Comment:
Renaming would make sense here, i suggest: documents and setDocuments for the State and response = useLoadDocuments(id) (without the .data, because we also need the .loading attribute)